packages/neon-core/src/sc/NativeContractMethodPrices.ts

```diff
@@ -0,0 +1,27 @@
+import { Fixed8 } from "../u";
+
+export const NativeContractMethodPrices = new Map<string, Fixed8>([
```

Comment:
any particular reason you chose a map instead of just a generic object?

Reply:
The keys are native contract methods, which are strings. Keys of enum and generic object are usually fixed prop name, and better in upper case. That's why I chose Map.